### ISSUE TITLE
App Engine testrunner doesn't behave like docs suggest

### ISSUE THREAD

**Issue by maltem-za** [GoogleCloudPlatform/python-docs-samples#500]
This is relatively minor - just something I noticed relating to [/appengine/standard/localtesting/runner.py](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/appengine/standard/localtesting/runner.py)

The App Engine [Setting up a testing framework](https://cloud.google.com/appengine/docs/python/tools/localunittesting#Python_Setting_up_a_testing_framework) documentation states: "Test files follow the convention of having **test** prefixed to their name."

However, 1fe6ae4 changed the behaviour to look for files **ending** with **_test.py**

This makes sense if you're trying to run the sample tests, but not if you've used the testrunner on another project and named your test modules according to what has up until now been the default expectation, being test files with test **prefixed**. If you were to update your testrunner code to the latest and greatest you'll now find none of your tests being run. That is **if** you notice - assuming most people don't set up their builds to fail on:

```
----------------------------------------------------------------------
Ran 0 tests in 0.000s

OK
```

So, I would argue that it would be better to stick with the default (discover test files prefixed with test) and instead rename the sample tests.

TL;DR;
minor mismatch with documentation, has potential to cause havoc for people that have used the testrunner and who update it

**Comment by jonparrott** [comment]
This is a great bug report. Thank you, I'll investigate shortly.
